avm/res/network/trafficmanagerprofile/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.29.47.4906",
-      "templateHash": "2090813965996228671"
+      "version": "0.28.1.47646",
```

Comment:
Please don't downgrade the bicep cli used to compile the template

Reply:
Rebuilt main.json with the latest bicep cli version